### ISSUE TITLE
LS: Add option to disable `cairo-test` plugin

### ISSUE THREAD

**Issue by Draggu** [starkware-libs/cairo#6552]
Currently there is no way to disable this. This should be disabled when using `snforge`. Maybe we can do it automatically by checking deps.

**Comment by mkaput** [comment]
yes, we should do this by checking deps, but only after stabilizing procmacros

**Comment by Arcticae** [comment]
Closing in favour of #6551
